internal/pkg/manifest/workload.go

```diff
@@ -441,14 +441,78 @@ func (p *PlacementArgs) isEmpty() bool {
 // PlacementString represents what types of subnets (public or private subnets) to place tasks.
 type PlacementString string
 
+// SecurityGroupsIDsOrConfig represents security groups attached to task. It supports unmarshalling
+// yaml which can either be of type SecurityGroupsConfig or a list of strings.
+type SecurityGroupsIDsOrConfig struct {
+	SecurityGroupIds     []string `yaml:"security_groups"`
```

Comment:
Do we need the tag? Since we already have `"security_groups"` YAML tag for `SecurityGroupsIDsOrConfig` I worried this would require a manifest of
```
network:
  vpc:
    security_groups:
      security_groups: "..."
```
for it to unmarshal.

Reply:
What do you think of renaming the fields to
```
type SecurityGroupsIDsOrConfig struct {
  IDs []string
  AdvancedConfig SecurityGroupsConfig
}

type vpcConfig struct {
  SecurityGroups SecurityGroupsIDOrConfig
}
```
so that in callers it reads
```
v := &vpcConfig{...}
ids := v.SecurityGroups.ID
config := v.SecurityGroups.AdvancedConfig
```

Reply:
suggested change sounds better. They are changed as suggested.